tests/www/test_views.py

```diff
@@ -1744,12 +1744,23 @@ def setUp(self):
         super().setUp()
         self.session = Session()
         models.DagBag().get_dag("example_bash_operator").sync_to_db(session=self.session)
+        self.graph_endpoint = '/graph?dag_id=example_bash_operator'
+        self.trigger_url = 'origin=%2F{}%3Fdag_id%3Dexample_bash_operator'
 
     def test_trigger_dag_button_normal_exist(self):
         resp = self.client.get('/', follow_redirects=True)
         self.assertIn('/trigger?dag_id=example_bash_operator', resp.data.decode('utf-8'))
         self.assertIn("return confirmDeleteDag(this, 'example_bash_operator')", resp.data.decode('utf-8'))
 
+    def test_trigger_dag_default_view(self):
+        resp = self.client.get(self.graph_endpoint, follow_redirects=False)
+        self.assertIn(self.trigger_url.format('tree'), resp.data.decode('utf-8'))
+
+    def test_trigger_dag_graph_view(self):
+        conf.set("webserver", "dag_default_view", "graph")
```

Comment:
```suggestion
```

It is safer to use the conf context manager:
```python
with conf_vars({('webserver', 'dag_default_view'): 'graph'}):
    ...
```
so that value will be reset afterwards.
You can also use it as function decorator `@conf_vars({('webserver', 'dag_default_view'): 'graph'})` which in my opinion is even cleaner.